### DISCUSSION TITLE
In which way I can remove all tailwdin styles for specific component In React?

### DISCUSSION THREAD

**Discussion by vanaf25** [tailwindlabs/tailwindcss#13935] [Help]
I have app.css. Code:@tailwind base;
@tailwind components;
@tailwind utilities;
In which way I can provide this styles only on specific components in React?

**Comment by AvaniBaheti** [comment]
You can create a specific css module for each of your react component and can add whatever styles you want for that particular file. Import css module file in your component ( import styles from './abc.module.css'.)
